mountpoint-s3/scripts/fs_bench.sh

```diff
@@ -75,11 +86,19 @@ read_benchmark () {
   for job_file in "${jobs_dir}"/*.fio; do
     mount_dir=$(mktemp -d /tmp/fio-XXXXXXXXXXXX)
 
+    job_name=$(basename "${job_file}")
+    job_name="${job_name%.*}"
+    log_dir=logs/${job_name}
+    rm -rf ${log_dir}
```

Comment:
Why are we removing the log files for every read job? Same question for write.

Reply:
For this PR, I also enable debug logs, that means the log files will be pretty big. I thought we probably run into "no space left" kind of error and want to avoid that.